mikkoaf/pilot-configurator
# Dev/nh

Rework view results page template to check if user is school or company and improve the code. The page works differently to create a list of company matches to a school user and to create a list of school matches for a company user.

Add additional admin tool to view school results in admin page so that admin can also check a school's result page.

Add new splash page for new users without school, company or administrator role. Moved and renamed the splash page code (splash_creation/create_splash.php) to pages/splashes.php.

Add templates to restrict pages to either school or company users and add these templates to company and school splash pages. (admin can also view pages with these templates)


Test suggestions:
- Check that view_results works for companies and schools when logged in as companies or schools.
- Check that previous also works as administrator through admin page's tools
- Log in as a user that isn't school, company or admin to see if you land on the new user splash
- Try to access school and company splashes on a wrong user
